packages/components/src/Notification/__snapshots__/Notification.snap.test.js.snap

```diff
@@ -18,6 +18,7 @@ exports[`Notification should render 1`] = `
       role="status"
     >
       <button
+        aria-label=""
```

Comment:
those are not supposed to be empty right ?

Reply:
This should be fixed in Notification component --> another PR
This PR is focused on Actions.